pkg/cmd/roachtest/tests/mixed_version_ldr.go

```diff
@@ -75,6 +75,7 @@ func InitLDRMixed(
 		mixedversion.NumUpgrades(expectedMajorUpgrades),
 		mixedversion.WithTag("left"),
 		mixedversion.WithSkipVersionProbability(0),
+		mixedversion.DisableAllFailureInjectionMutators(),
```

Comment:
Failure injections aren't compatible with these ldr tests but I'm not 100% what a proper reasoning to put for documentation would be, if there are any suggestions that would be great!